dev/lib/product_taxonomy/cli.rb

```diff
@@ -5,7 +5,13 @@ class Cli < Thor
     desc "dist", "Generate the taxonomy distribution"
     def dist
       seconds = Benchmark.realtime do
-        Value.load_from_source(source_data: YAML.safe_load_file("../data/values.yml"))
+        source_data = YAML.safe_load_file("../data/attributes.yml")
```

Comment:
Minor nitpick:

Why do we load this before the line below? It made me think we need this before we load values.

Also should we give this a different name since we're now loading multiple "models"?

Reply:
You're right. I'm using this command as an easy entrypoint into the program for now, mainly so I can keep an eye on performance/runtime. Until the `dist` command is actually implemented, whatever is here can be considered scratch work/temporary.

But maybe it would be less confusing to move it somewhere else, like a test case without any assertions that just measures runtime.